### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by BcnCarlos**
https://auth0team.atlassian.net/jira/servicedesk/projects/DR/issues/DR-2050?filter=allissues

Minor edit to the ClientID.

### CODE REVIEW

**Review by avanscoy — DISMISSED** [review]
Nice work!

File: articles/quickstart/native/maui/interactive.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ using Auth0.OidcClient;
 var client = new Auth0Client(new Auth0ClientOptions
 {
     Domain = "${account.namespace}",
-    ClientId = "${account.namespace}",
+    ClientId = "{yourClientID}",
```
Comment (frederikprijck) [inline articles/quickstart/native/maui/interactive.md]:
```suggestion
    ClientId = ${account.clientId}
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: config/redirects.js [hunk 2]
```diff
@@ -83,6 +83,10 @@ const redirects = [
       `/quickstart/native/:platform/${apis}`,
     ],
     to: '/quickstart/native/:platform',
+  },
+   {
+    from: `/quickstart/native/maui`,
+    to: '/quickstart/native/maui/interactive',
```
Comment (frederikprijck) [inline config/redirects.js]: We should not do this, as the non interactive version is still used in the dashboard (manage.auth0.com).

The docs platform should have a general redirect in place to redirect to the interactive when not running in manage.auth0.com.

If there is an issue with those redirects, that should be addressed at that level and not enforce a global redirect.

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: articles/quickstart/native/maui/interactive.md (hunk 1 above)
Comment (frederikprijck) [reply articles/quickstart/native/maui/interactive.md]: Good catch, we should not use namespace.

However, we do need account.clientId as that is auto populated when reading through the quickstart while logged in.

Comment (frederikprijck) [reply articles/quickstart/native/maui/interactive.md]: When not logged in, these values will be replaced with other placeholders, such as yourDomain and yourClientId.

File: articles/quickstart/native/maui/interactive.md [hunk 3]
```diff
@@ -105,7 +105,7 @@ using Auth0.OidcClient;
 var client = new Auth0Client(new Auth0ClientOptions
 {
     Domain = "${account.namespace}",
-    ClientId = "${account.namespace}",
+    ClientId = ${account.clientId}
```
Comment (frederikprijck) [inline articles/quickstart/native/maui/interactive.md]:
```suggestion
    ClientId = "${account.clientId}",
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: articles/quickstart/native/maui/interactive.md (hunk 3 above)
Comment (frederikprijck) [reply articles/quickstart/native/maui/interactive.md]: Sorry that was my bad :(

**Review by frederikprijck — APPROVED** [review]

**Review by BcnCarlos — COMMENTED** [review]
updated ClientId variable

**Comment by frederikprijck** [comment]
Thanks for fixing that!

**Pull request merged**